hack/lib/test/junit.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# This utility file contains functions that format test output to be parsed into jUnit XML
+
+# os::test::junit::declare_suite_start prints a message declaring the start of a test suite
+# Any number of suites can be in flight at any time, so there is no failure condition for this
+# script based on the number of suites in flight.
+#
+# Globals:
+#  - JUNIT_REPORT_OUTPUT
+#  - NUM_SUITES_IN_FLIGHT
+# Arguments:
+#  - 1: the suite name that is starting
+# Returns:
+#  - increment NUM_SUITES_IN_FLIGHT
+function os::test::junit::declare_suite_start() {
+    local suite_name=$1
+    local num_suites=${NUM_SUITES_IN_FLIGHT:-0}
+    
+    echo "==== BEGIN TEST SUITE github.com/openshift/origin/test/${suite_name} =====" >> "${JUNIT_REPORT_OUTPUT}"
```

Comment:
The fact that `github.com/openshift/origin/test` is hardcoded suggests that its a zero entropy statement and therefore worthless.  Why do we have it?